.gitignore

```diff
@@ -11,6 +11,7 @@ MANIFEST
 .pytest_cache/
 .ipynb_checkpoints/
 .vscode/
+result_images/
```

Comment:
One rule in the [contributing guidelines](https://github.com/GenericMappingTools/pygmt/blob/master/CONTRIBUTING.md#general-guidelines) is:

> Do not commit changes to files that are irrelevant to your feature or bugfix (eg: .gitignore, IDE project files, etc).

Perhaps better to open a separate PR for the changes in `.gitignore` and `Makefile`?

Reply:
Ok, will cherry pick that commit into a separate PR.